src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -175,7 +179,8 @@ Namespace Microsoft.CodeAnalysis.Semantics
                 Dim syntax As SyntaxNode = If(value.Syntax?.Parent, expression.Syntax)
                 Dim type As ITypeSymbol = target.Type
                 Dim constantValue As [Optional](Of Object) = value.ConstantValue
-                Dim assignment = New SimpleAssignmentExpression(target, value, _semanticModel, syntax, type, constantValue)
+                Dim isImplicit As Boolean = expression.WasCompilerGenerated
```

Comment:
>Dim isImplicit As Boolean = expression.WasCompilerGenerated [](start = 16, length = 59)

This looks suspicious, probably "implicitness" of the assignment node should be based off of the "implicitness" of ```value```.